### PR TITLE
[WIP] Added API test for KubeVirt

### PR DESCRIPTION
**Pull request by vatsalparekh**
Added API tests for KubeVirt

### CODE REVIEW

File: robottelo/config/base.py [hunk 1]
```diff
@@ -599,6 +599,23 @@ def validate(self):
         return validation_errors
 
 
+class KubeVirtSettings(FeatureSettings):
+    "KubeVirt host settings definations"
```
Comment (ntkathole) [inline robottelo/config/base.py]: s/definations/definitions

File: tests/foreman/api/test_computeresource_kubevirt.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
+# -*- encoding: utf-8 -*-
+"""Unit tests for the ``compute_resource`` paths.
+
+A full API reference for compute resources can be found here:
+http://www.katello.org/docs/api/apidoc/compute_resources.html
+
+
+:Requirement: Computeresource Libvirt
```
Comment (ntkathole) [inline tests/foreman/api/test_computeresource_kubevirt.py]: you mean Kubevirt?

Comment (vatsalparekh) [reply tests/foreman/api/test_computeresource_kubevirt.py]: copy paste typo

**Comment by vijay8451** [comment]
@vatsalparekh thanks for the PR :)

**Comment by jyejare** [comment]
@vatsalparekh Would you like to revisit this or would like to someone from SatelliteQE to take up ?

**Comment by JacobCallahan** [comment]
@SatelliteQE/quality-engineers does anyone want to take this up? If not, this is a candidate for closure.

**Pull request closed without merging**